### ISSUE TITLE
Add context page for API docs

### ISSUE THREAD

**Issue by harinikmsft** [microsoft/react-native-windows-samples#103]
It would be good to add a page that sets context on what the API docs section is doing - point to RN docs for all RN APIs, set context on Windows specific work, Mac specific work etc., Right now it is confusing to land in that section without context.

**Comment by chrisglein** [comment]
Agreed. It pretty unceremoniously drops you into a specific API: https://microsoft.github.io/react-native-windows/docs/flyout-component
Which is what the Facebook docs do, but theirs are more comprehensive.

**Comment by harinikmsft** [comment]
There is a good context page for the [react native API docs](https://reactnative.dev/docs/components-and-apis) now.

**Comment by crutchcorn** [comment]
I'd be happy to write this if that would be helpful
